sass/layout/_sidebar-left.scss

```diff
@@ -981,6 +980,7 @@
     .SidebarChannelLinkLabel_wrapper {
         overflow: hidden;
         display: flex;
+        margin-right: auto;
```

Comment:
Now `.SidebarChannelLinkLabel_wrapper` pushes `.badge` and `. SidebarMenu` rightwards, this frees up margin-left on .badge to be used for "padding"